src/quickgui/qgsquickpositionkit.h

```diff
@@ -65,6 +76,16 @@ class QUICK_EXPORT QgsQuickPositionKit : public QObject
      */
     Q_PROPERTY( bool isSimulated READ simulated NOTIFY isSimulatedChanged )
 
+    /**
+     * Associated map settings. Should be initialized before the first use from mapcanvas map settings.
+     */
+    Q_PROPERTY( QgsQuickMapSettings *mapSettings MEMBER mMapSettings WRITE setMapSettings NOTIFY mapSettingsChanged )
+
+
+    Q_PROPERTY( QVector<double> simulatePositionLongLatRad READ simulatePositionLongLatRad WRITE setSimulatePositionLongLatRad NOTIFY simulatePositionLongLatRadChanged )
```

Comment:
as it is debug tool, I am wondering if it is not better to just have this as QString
`pos = "23.222, -30.000, 60"`

and in c++ we would do string.split(',') and if size != 3 do nothing?